pkg/apis/kubeone/config/config.go

```diff
@@ -42,7 +42,8 @@ import (
 
 const (
 	// KubeOneClusterKind is kind of the KubeOneCluster object
-	KubeOneClusterKind = "KubeOneCluster"
+	KubeOneClusterKind                  = "KubeOneCluster"
+	flagsAndFeatureGateOverridesWarning = "%s only covers %s. For some features, you may also need additional configuration for other components. Use this at your own risk!!"
```

Comment:
I think we could have more explicit message, let me see if I can come up with something.